lua/neo-tree/defaults.lua

```diff
@@ -491,6 +493,61 @@ local config = {
       },
     },
   },
+  document_symbols = {
+    renderers = {
+      root = {
+        {"indent"},
+        {"icon", default="C" },
+        {"name", zindex = 10},
+      },
+      symbol = {
+        {"indent"},
+        {"icon", default="C" },
+        {"container",
+        content = {
+          {"name", zindex = 10},
+          {"kind_name", zindex = 20, align = "right"},
+          {"kind_icon", zindex = 20, align = "right"},
+          }
+        }
+      },
+    },
+    window = {
+      mappings = {
+        ["<cr>"] = "toggle_node",
+        ["o"] = "jump_to_symbol",
```

Comment:
I think it is more natural for `<cr>` to be mapped to `jump_to_symbol`, that follows the convention of the rest of the sources. `toggle_node` should be mapped to `<space>` by default.